MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1570,7 +1593,15 @@ public string GetGraphicalSubtitleParam(EncodingJobInfo state, EncodingOptions o
                     state.VideoStream.Width.Value,
                     state.VideoStream.Height.Value);
 
-                videoSizeParam += ":force_original_aspect_ratio=decrease";
+                if (string.Equals(outputVideoCodec, "h264_qsv", StringComparison.OrdinalIgnoreCase))
+                {
+                    //For QSV, feed it into hardware encoder
+                    videoSizeParam += ",hwupload=extra_hw_frames=64";
+                } 
+                else 
+                {
+                    videoSizeParam += ":force_original_aspect_ratio=decrease";
```

Comment:
this changes the non-`qsv` behaviour, why do you want to stuff it in QSV-related PR?

Reply:
force_original_aspect_ratio=decrease was the original code. i've now removed this entirely because it actually causes the bitmap subtitles to appear off-centre.